norent/schema.py

```diff
@@ -242,7 +242,23 @@ def perform_mutate(cls, form, info: ResolveInfo):
 @schema_registry.register_mutation
 class NorentFullName(NorentScaffoldingOrUserDataMutation):
     class Meta:
-        form_class = forms.FullName
+        form_class = forms.FullLegalName
```

Comment:
Is it OK that this is FullLegalName?

Reply:
It might be, but I'm not actually sure, due to the way that our Python code gets turned into a GraphQL schema on-the-fly.

I recommend seeing if anything called `FullName` ends up being changed to `FullLegalName` in our `schema.json` file!

Reply:
Oh, also, in general the entire chunk of `schema.json` that refers to the `norentFullName` mutation should basically be unchanged, with the sole exception of `deprecation_reason`!  So that 's another way of determining whether this changed anything to be backwards-incompatible.

Reply:
Er, so yet _another_ way of verifying your assumptions would be to make a trivial subclass of `FullLegalName` called `BoopBoopBoop`, assign `form_class` to it, and see if "boop" appears anywhere in `schema.json`.

Reply:
OK! I checked `schema.json` and I the code related to `norentFullName` is indeed unchanged - we just add more code to handle `noRentFullLegalName`.

I also subclassed `FullLegalName` to `Doop` (artistic license, assigned `form_class` here to it and regenerated `schema.json` - nothing changed in the `schema.json`, so this seems safe.